README.md

```diff
@@ -1,23 +1,31 @@
 # Normal OJ
 
-## How to
+## Clone this project
 
-1. `git clone --recurse-submodules https://github.com/Normal-OJ/Normal-OJ.git`
-2. `cd Normal-OJ`
-3. `git submodule foreach --recursive git checkout master`
+```bash
+git clone https://github.com/Normal-OJ/Normal-OJ.git
+```
+Enter the project directory:
+```bash
+cd Normal-OJ
+```
+Clone submodules:
+```bash
+git submodule update --init --recursive
+git submodule foreach --recursive git checkout main
+```
 
-### Push
+## Overview of the project
+Checkout the [Introduction](https://github.com/Normal-OJ).
 
-For example, if you want to push `Normal-OJ/Back-End`:
+NOJ contains three parts:
+1. **Backend**: Python web server, provides RESTful API, communicates with the database and the sandbox.
+2. **Frontend**: The user interface that interacts with the backend, written in Vue.js.
+3. **Sandbox**: Takes the user's submission, compiles and executes the code, and returns the result to the backend.
```

Comment:
```suggestion
1. **[Backend](https://github.com/Normal-OJ/Back-End)**: Python web server, provides RESTful API, communicates with the database and the sandbox.
2. **[Frontend](https://github.com/Normal-OJ/new-front-end)**: The user interface that interacts with the backend, written in Vue.js.
3. **[Sandbox](https://github.com/Normal-OJ/Sandbox)**: Takes the user's submission, compiles and executes the code, and returns the result to the backend.
```
Feel like it's worth to add repo links here. In case they find the wrong repo like v1 frontend. (though they can find it through submodule still)